### DISCUSSION TITLE
Disable dependabot?

### DISCUSSION THREAD

**Discussion by AstroSquared** [community/community#22369] [New to GitHub]
<p>It says it in the title.</p>

**Comment by ghost** [comment, marked as answer]
<p>You can disable Dependabot by going to Settings &gt; Security &amp; analysis<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/f/5f464aeb9b2fd15fedf94527068b64ec671e6b35.png" data-download-href="/uploads/short-url/dAPXA9f1Ewjtnw1ccTZ9HrsB0UJ.png?dl=1" title="Settings page" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/10137/181070589-d2fc6590-4bd6-4e2d-8b56-cddfb4bc1fb4.png" alt="Settings page" data-base62-sha1="dAPXA9f1Ewjtnw1ccTZ9HrsB0UJ" width="690" height="211" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/f/5f464aeb9b2fd15fedf94527068b64ec671e6b35_2_690x211.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/f/5f464aeb9b2fd15fedf94527068b64ec671e6b35_2_1035x316.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/f/5f464aeb9b2fd15fedf94527068b64ec671e6b35.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/f/5f464aeb9b2fd15fedf94527068b64ec671e6b35_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Settings page</span><span class="informations">1324×405 36.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ssbarnea** [comment]
<p>As it seems that there are no signs dependabot spam will go away, I ended up disabling it and avoiding adopting it for any new projects. <a href="https://github.com/ansible-community/ansible-lint/pull/1396" class="inline-onebox" rel="noopener nofollow ugc">Disable dependabot due to spammy nature by ssbarnea · Pull Request #1396 · ansible-community/ansible-lint · GitHub</a></p>
<p>Dependabot is great idea, but it fails to cover some critical aspects like being able to create single update commit with all updated dependencies. Lack of this feature makes is impractical for any projects that have either many dependencies or some that change often. Usually on python world, this means &gt;10 deps in total, it already became a burden.</p>

**Comment by aaronfranke** [comment]
<p>I already have Dependabot disabled in the Security &amp; analysis page (as shown by the first below image), but I still receive unwanted PRs from Dependabot (as shown by the second below image). How do I disable them? This is on a fork, so I <em><strong>never</strong></em> want these PRs, instead I will let the upstream repo update their dependencies and then I will fast-forward to their master.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/a/7abc502c50738f8c679bac2bff41d3d13774e676.png" data-download-href="/uploads/short-url/hvLI3dnU88hqEGoAoprpXkVFgxM.png?dl=1" title="Screenshot from 2021-05-13 05-41-36" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1646875/181070591-f3527b90-0b83-4752-b9c4-ae1a013fd124.png" alt="Screenshot from 2021-05-13 05-41-36" data-base62-sha1="hvLI3dnU88hqEGoAoprpXkVFgxM" width="690" height="290" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/a/7abc502c50738f8c679bac2bff41d3d13774e676_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2021-05-13 05-41-36</span><span class="informations">950×400 27.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/4/24b277630ce74d1040e52bfa9d1b94afb8ad6348.png" data-download-href="/uploads/short-url/5eDyblqCX7LZ3VmwnCdtVZAPBkk.png?dl=1" title="Screenshot from 2021-05-13 05-45-38" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1646875/181070592-53063a25-a087-4749-ac4c-3109a788d586.png" alt="Screenshot from 2021-05-13 05-45-38" data-base62-sha1="5eDyblqCX7LZ3VmwnCdtVZAPBkk" width="555" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/4/24b277630ce74d1040e52bfa9d1b94afb8ad6348_2_555x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/4/24b277630ce74d1040e52bfa9d1b94afb8ad6348_2_832x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/4/24b277630ce74d1040e52bfa9d1b94afb8ad6348.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/4/24b277630ce74d1040e52bfa9d1b94afb8ad6348_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2021-05-13 05-45-38</span><span class="informations">1000×900 82.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by TheLastGimbus** [comment]
<p>I had this too, and I forgot that dependabot is initially enabled by adding a <code>.github/dependabot.yaml</code> file - just delete it :+1:</p>

**Reply by bpiv400** [reply]
Did you just delete this on your main/master?

**Comment by huan** [comment]
<p>For NPM, dependabot always check <code>@next</code> which is really annoying.</p>
<p>I search a lot and try to disable the <code>@next</code> channel updates, but with no luck, and find nothing. So I have to disable the whole dependabot to help myself stop being spamming by it.</p>
<p>I really miss greenkeeper more, the greenkeeper is really better than the dependabot on GitHub today.</p>

**Comment by aaronfranke** [comment]
<p><a class="mention" href="/thelastgimbus">@TheLastGimbus</a> In my situation, I get dependabot PRs on a fork of another repository. Deleting a file in the repo is not an option because I want to keep my master branch in sync with the upstream one.</p>

**Comment by FranklinYu** [comment]
<p>I got the idea from <a href="https://github.com/twbs/bootstrap/discussions/31385#discussioncomment-45054" rel="noopener nofollow ugc">Bootstrap folks</a>: you can delete them in another branch (like <code>master-2</code>), then set that branch as the default branch.</p>

**Comment by quantuminformation** [comment]
i hate the dependabot spam, i dont have any of the checkbox ticked and it still halo-floods my notifications
